WordPress/Classes/ViewRelated/Reader/ReaderBlockedSiteCell.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+ public class ReaderBlockedSiteCell: UITableViewCell
```

Comment:
Potentially a silly question: We don't need 600pt width capping on this one right?